src/services/auth.js

```diff
@@ -34,7 +36,7 @@ const authService = {
       throw createError(401, USER_NOT_FOUND)
     }
 
-    const checkedPassword = (password === user.password) || isFromGoogle
+    const checkedPassword = password === user.password || isFromGoogle
```

Comment:
What about priorities? I think it would be better this way.
```suggestion
    const checkedPassword = isFromGoogle || password === user.password;
```